pvacseq/lib/main.py

```diff
@@ -77,40 +89,42 @@ def main(args_input = sys.argv[1:]):
     )
     print("Completed")
 
-    netmhc_output = set(
-        run([args.netmhc_path+" -A"],
-            shell=True, stdout=PIPE
-            ).stdout.decode().split('\n')
-        )
+#    netmhc_output = set(
+#        run([args.netmhc_path+" -A"],
+#            shell=True, stdout=PIPE
+#            ).stdout.decode().split('\n')
+#        )
 
-    for epl in args.epitope_length:
-        for a in args.allele:
-            if a in netmhc_output:
-                net_out = ".".join([args.sample_name, a, str(epl), "netmhc.txt"])
-                print("Running NetMHC on Allele", a,
-                      "and Epitope Length", epl
-                      )
-                netmhc_cmd = "%s -a %s -l %d %s -x %s" % (
-                    args.netmhc_path, a, epl,
+    iedb_output_files = []
+    for method in args.prediction_algorithms:
+        iedb_method = prediction_method_lookup(method)
+        for epl in args.epitope_length:
+            for a in args.allele:
+ #               if a in netmhc_output:
```

Comment:
Lines 92-96 had functionality to look up the available list of alleles from NetMHC. This was then checked on this line against the list of alleles input by the user. IEDB has it's own post request for looking up available alleles. Unfortunately, that is currently broken. I put in a request to have it fixed. Once it is available we should incorporate this check again. I'm not sure if we should delete the commented out code until then.